docs/content/Upgrade/manual/upgrade35.md

```diff
@@ -50,7 +49,7 @@ For example, given the following output:
 ```
 $ pgo show cluster mycluster
 
-cluster : mycluster (crunchy-postgres:centos7-11.5-2.4.2)
+cluster : mycluster (crunchy-postgres:{{< param centosBase >}}-11.5-2.4.2)
```

Comment:
This output is from a specific (older) version. Should it stay fixed to `centos7`?